src/core/osmLayer.js

```diff
@@ -55,6 +59,10 @@ geo.osmLayer = function (arg) {
     m_imageFormat = arg.imageFormat;
   }
 
+  if (arg && arg.displayLast !== undefined) {
```

Comment:
Sure.. you want to check if this check (arg.displayLast) results to true? (arg && arg.displayLast !== undefined && arg.displayLast. So does it mean that arg.displayLast could result to false as well? Just wanted to make sure I understand your intentions.